relay-server/src/services/processor/span/processing.rs

```diff
@@ -148,6 +148,13 @@ pub fn extract_from_event(state: &mut ProcessEnvelopeState<TransactionGroup>) {
         return;
     };
 
+    if !state
+        .project_state
+        .has_feature(Feature::SpanMetricsExtraction)
```

Comment:
We should document the `SpanMetricsExtraction` variant to reflect that it toggles both spans and span metrics. Maybe even rename the variant (but not the string representation).